mpas_analysis/test/test_generalized_reader.py

```diff
@@ -43,6 +43,7 @@ def test_variableMap(self):
             ds = open_multifile_dataset(
                 fileNames=fileName,
                 calendar=calendar,
+                config=None,
```

Comment:
For now, config isn't used to I'm just passing `None`.  If we start making used of `config`, one will need to be set up here to serve the necessary options (e.g. chunk sizes).